test/benches/bench.rs

```diff
@@ -3,8 +3,54 @@ use criterion::Throughput;
 use criterion::{criterion_group, criterion_main, Criterion};
 use holochain_wasmer_host::prelude::*;
 use rand::prelude::*;
+use tempfile::TempDir;
 use test::wasms::TestWasm;
 use wasmer::AsStoreMut;
+use wasmer::Module;
+use wasmer::Store;
+
+/// compile a module
+pub fn wasm_module_compile(c: &mut Criterion) {
```

Comment:
Where are these benchmark results kept? I'd expect the results to be in source control so that differences can be tracked. Or some other method of knowing what the previous results were when making changes

Reply:
@ThetaSinner we used to have a whole benchmarking server but it was decommissioned as it was too verbose and we never reinstated anything